packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -342,12 +396,17 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
     // If the notification.dragDetails is null, this scroll is not triggered by
     // user dragging. It may be a result of ScrollController.jumpTo or ballistic scroll.
     // In this case, we don't want to trigger the refresh indicator.
-    return ((notification is ScrollStartNotification && notification.dragDetails != null)
-            || (notification is ScrollUpdateNotification && notification.dragDetails != null && widget.triggerMode == RefreshIndicatorTriggerMode.anywhere))
-      && (( notification.metrics.axisDirection == AxisDirection.up && notification.metrics.extentAfter == 0.0)
-            || (notification.metrics.axisDirection == AxisDirection.down && notification.metrics.extentBefore == 0.0))
-      && _mode == null
-      && _start(notification.metrics.axisDirection);
+    return ((notification is ScrollStartNotification &&
+                notification.dragDetails != null) ||
+            (notification is ScrollUpdateNotification &&
+                notification.dragDetails != null &&
+                widget.triggerMode == RefreshIndicatorTriggerMode.anywhere)) &&
+        ((notification.metrics.axisDirection == AxisDirection.up &&
+                notification.metrics.extentAfter == 0.0) ||
+            (notification.metrics.axisDirection == AxisDirection.down &&
+                notification.metrics.extentBefore == 0.0)) &&
+        _mode == null &&
+        _start(notification.metrics.axisDirection);
```

Comment:
Prefer the previous code, as per the style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#align-expressions